Shared/Samples/Change viewpoint/ChangeViewpointView.swift

```diff
@@ -0,0 +1,127 @@
+                            }
+                        case .geometry:
+                            Task {
+                                try await mapViewProxy.setViewpointGeometry(
+                                    model.griffithParkGeometry,
+                                    padding: 50
+                                )
+                            }
+                        case .animate:
+                            Task {
+                                if let center = visibleArea?.extent.center {
+                                    try await mapViewProxy.setViewpoint(
+                                        Viewpoint(center: center, scale: model.scale / 2),
+                                        duration: 5
+                                    )
+                                }
+                            }
+                        }
+                    }
+                HStack {
+                    Picker("Viewpoint Type", selection: $model.viewpointType) {
+                        ForEach(ViewpointType.allCases, id: \.self) { mode in
+                            Text(mode.label)
+                        }
+                    }
+                    .pickerStyle(.segmented)
+                    .frame(maxWidth: 540)
+                    .padding()
+                }
+            }
+        }
+    }
+}
+
+extension ChangeViewpointView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    class Model: ObservableObject {
+        /// A map with an imagery basemap style.
+        let map = Map(basemapStyle: .arcGISImagery)
+        
+        /// The scale of the map view.
+        var scale: Double = 0
+
+        /// The current viewpoint type.
+        @Published var viewpointType: ViewpointType = .geometry
+        
+        /// A polygon representing the area of Griffith Park.
+        var griffithParkGeometry: ArcGIS.Polygon!
+        
+        /// A  point representing the location of London.
+        var londonCoordinate: Point = {
+            return Point(x: 0.1275, y: 51.5072, spatialReference: .wgs84)
+        }()
+    
+        /// The graphics overlays containing a graphic.
+        var graphicsOverlays = GraphicsOverlay()
+        
+        init() {
+            let geometry = try? Geometry.fromJSON(Self.griffithParkJsonString)
+            griffithParkGeometry = geometry as? ArcGIS.Polygon
+            let griffithParkSymbol = SimpleFillSymbol(
+                style: .solid,
+                color: UIColor(red: 0, green: 0.5, blue: 0, alpha: 0.7)
+            )
+            let griffithParkGraphic = Graphic(
+                geometry: griffithParkGeometry,
+                symbol: griffithParkSymbol
+            )
+            graphicsOverlays.addGraphic(griffithParkGraphic)
+        }
+        
+        /// A JSON string for the simulated path.
+        private static var griffithParkJsonString: String {
```

Comment:
Can you move this variable to the bottom of the file? We typically keep the GeoJSON at the bottom to avoid obstructing other code.